modules/widgets/wordpress-post-widget.php

```diff
@@ -151,6 +155,12 @@ public function form( $instance ) {
 			$number_of_posts = 5;
 		}
 
+		if ( isset( $instance[ 'open_in_new_window'] ) ) {
+			$open_in_new_window = $instance[ 'open_in_new_window'];
+		} else {
+			$open_in_new_window = false;
+		}
```

Comment:
@georgestephanis, what's the preferred code style here?

Personally, I would probably do:
```php
$open_in_new_window = false;
if ( isset( $instance[ 'open_in_new_window'] ) ) {
	$open_in_new_window = $instance[ 'open_in_new_window'];
}
```